Mathlib/Analysis/SpecialFunctions/NonIntegrable.lean

```diff
@@ -15,25 +15,25 @@ import Mathlib.MeasureTheory.Integral.FundThmCalculus
 # Non integrable functions
 
 In this file we prove that the derivative of a function that tends to infinity is not interval
-integrable, see `interval_integral.not_integrable_has_deriv_at_of_tendsto_norm_at_top_filter` and
-`interval_integral.not_integrable_has_deriv_at_of_tendsto_norm_at_top_punctured`.  Then we apply the
-latter lemma to prove that the function `λ x, x⁻¹` is integrable on `a..b` if and only if `a = b` or
-`0 ∉ [a, b]`.
+integrable, see `not_intervalIntegrable_of_tendsto_norm_atTop_of_deriv_isBigO_filter` and
+`not_intervalIntegrable_of_tendsto_norm_atTop_of_deriv_isBigO_punctured`. Then we apply the
+latter lemma to prove that the function `fun x => x⁻¹` is integrable on `a..b` if and only if
+`a = b` or `0 ∉ [[a, b]]`.
```

Comment:
Why have you added double square brackets here? Probably should be reverted. (And below.)

Reply:
https://github.com/leanprover-community/mathlib4/blob/2bb9fc30b4032413c72f9700462b74e23fa0acae/Mathlib/Data/Set/Intervals/UnorderedInterval.lean#L59C6-L63